build/ci/github-actions/examples/isolated-builders.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Isolated builders
+description:
+keywords:
+---
+
```

Comment:
Hum we don't have any explanations here. I think we could say that this workflow can be useful to isolate subsequent builds in your workflow.